pkg/virt-handler/container-disk/mount.go

```diff
@@ -247,8 +248,7 @@ func (m *mounter) Mount(vmi *v1.VirtualMachineInstance, verify bool) error {
 				f.Close()
 
 				log.DefaultLogger().Object(vmi).Infof("Bind mounting container disk at %s to %s", strings.TrimPrefix(sourceFile, nodeRes.MountRoot()), targetFile)
-				// #nosec g204 no risk to TrimPref as argument as it just trims two fixed strings
```

Comment:
Did you check whether the gosec will popup again here?

Reply:
Yes :) Doubled checked thanks to your comment.